Sparse_and_dense.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from scipy import sparse\n",
+    "import sys\n",
+    "import os\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.random.seed(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sparse .npz (compressed) takes the least disk space regardless of density\n",
+    "Uncompressed sparse matrices take more disk space when the matrices are very dense"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CCpGeAeGaD (1391, 137)\n",
+      "density: 97.8 %\n",
+      "dense: 1.525 MB\n",
+      "sparse: 1.347 MB\n",
+      "sparse_uncompressed: 2.238 MB\n",
```

Comment:
disambiguating the different storage formats can get a little confusing. Let's be as explicit as possible:

+ dense-uncompressed.npy
+ sparse-compressed.npz
+ sparse-uncompressed.npz

And my suggested file name would be to prepend `{metapath}-`